lib/get-config.js

```diff
@@ -51,7 +51,10 @@ module.exports = async (opts, logger) => {
     ...pickBy(options, option => !isUndefined(option) && !isNull(option)),
   };
 
-  options.repositoryUrl = options.repositoryUrl ? getGitAuthUrl(options.repositoryUrl) : options.repositoryUrl;
+  console.log(options.repositoryUrl, options.branch);
```

Comment:
do you want this `console.log` ? shouldn't be a `debug()` ?

Reply:
Oops that was for dev debugging and I forgot to remove it. Good catch!